h2/src/test/org/h2/test/scripts/testSimple.sql

```diff
@@ -1281,3 +1281,9 @@ select max(col1) from test;
 
 drop table if exists test;
 > ok
+
+drop table if exists test_link;
+> ok
+
+create linked table test_link('org.h2.Driver', 'jdbc:h2:mem:', 'sa', 'sa', 'DUAL') FETCH_SIZE 1000;
+> ok
```

Comment:
Please, don't use `testScript.sql` and `testSimple.sql` for new tests, these files are going to be removed from H2 when all their content will be moved out into better places.